### ISSUE TITLE
Publish Package to npm?

### ISSUE THREAD

**Issue by cycomachead** [IMSGlobal/caliper-js#199]
Are there intentions of publishing the package to npm? I realize it's not technically necessary to include this, but it's certainly easier than hunting for a package to find nothing. :) 

Thanks!

**Comment by arwhyte** [comment]
Yes, we have permission now from IMS to produce an npm package.  I'll work with IMS to see if we can get a packaged published in the next couple of weeks.

**Comment by ViktorHaag** [comment]
My understanding is that:
- publicly released versions of the sensor code can be put on package repos (not in development ones)
- IMS has to own the account that actually publishes the packages to repostories

This is material for the Python (and Java and .NET and Ruby) sensor as well, I assume?

**Comment by arwhyte** [comment]
@ViktorHaag -- yes, we would aim to add package publishing to our workflow for all sensors.   I spoke with @mgylling about this yesterday along with adding CI tooling to the sensors.  IMS supports both initiatives.  In the case of package publishing @LMattson has taken on the responsibility of securing accounts so that we can publish artifacts.  IMS is also reviewing CI tooling options.
